src/SwedbankPay.Sdk.Infrastructure/PaymentOrders/PaymentOrder.cs

```diff
@@ -0,0 +1,59 @@
+﻿using SwedbankPay.Sdk.Common;
+using System;
+using System.Globalization;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    public class PaymentOrder : IPaymentOrder
+    {
+        public PaymentOrder(PaymentOrderDto paymentOrder)
+        {
+            Id = new Uri(paymentOrder.Id, UriKind.RelativeOrAbsolute);
+            Amount = paymentOrder.Amount;
+            Created = paymentOrder.Created;
+            Currency = new CurrencyCode(paymentOrder.Currency);
+            CurrentPayment = paymentOrder.CurrentPayment.Map();
+            Description = paymentOrder.Description;
+            Language = new CultureInfo(paymentOrder.Language);
+            Metadata = paymentOrder.Metadata;
+            Operation = paymentOrder.Operation;
+            OrderItems = paymentOrder.OrderItems?.Map();
+            PayeeInfo = paymentOrder.PayeeInfo.Map();
+            Payers = paymentOrder.Payer.Map();
+            if(paymentOrder.Payments != null && string.IsNullOrEmpty(paymentOrder.Payments.Id))

```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)